arbitrum-docs/run-arbitrum-node/arbos-releases/arbos11.md

```diff
@@ -22,7 +22,7 @@ Formal release notes can be found [here](https://github.com/OffchainLabs/nitro/r
   - [EIP-3855: PUSH0 instruction](https://eips.ethereum.org/EIPS/eip-3855)
   - [EIP-3860: Limit and meter initcode](https://eips.ethereum.org/EIPS/eip-3860)
   - [EIP-6049: Deprecate SELFDESTRUCT](https://eips.ethereum.org/EIPS/eip-6049)
-- Improvements and fixes for [retryable tickets](../../../arbos/l1-to-l2-messaging.mdx) to ensure that the fee calculation to redeem retryable tickets will take into account both the infrastructure fee and the network fee. The infrastructure fee is the minimum L2 base fee only, while the network fee collects L2 congestion charges. This is important for [AnyTrust chains](../../../inside-anytrust.mdx) like Arbitrum Nova because members of the Data Availability Committee (DAC) gets paid a percentage of the infrastructure fee but not the network fee. Previously, the calculations to determine the fee for redeeming retryable tickets did not consider the infrastructure fee.
+- Improvements and fixes for [retryable tickets](/arbos/l1-to-l2-messaging.mdx) to ensure that the fee calculation to redeem retryable tickets will take into account both the infrastructure fee and the network fee. The infrastructure fee is the minimum L2 base fee only, while the network fee collects L2 congestion charges. This is important for [AnyTrust chains](/inside-anytrust.mdx) like Arbitrum Nova because members of the Data Availability Committee (DAC) gets paid a percentage of the infrastructure fee but not the network fee. Previously, the calculations to determine the fee for redeeming retryable tickets did not consider the infrastructure fee.
```

Comment:
There's some `.mdx` here; not sure if that's on purpose. Nonblocking, I suppose.

Reply:
Yep, those are mdx docs (for now). Ty!